extensions/resteasy-classic/rest-client/runtime/pom.xml

```diff
@@ -48,6 +48,10 @@
                     <groupId>org.jboss.weld</groupId>
                     <artifactId>weld-api</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>commons-logging</groupId>
+                    <artifactId>commons-logging</artifactId>
+                </exclusion>
```

Comment:
I wonder if we should add it to our exclusion list in the build parent?

Also maybe we should add the JBoss Logging Bridge? You should be able to find which one to use in some of the other extensions.

Reply:
Sure, I will give it a try

Reply:
@gsmet, `commons-logging` is already in the exclude list of the build parent (https://github.com/quarkusio/quarkus/blob/992dc5dd8701cfe412ae8e323ccb38ae00829e18/build-parent/pom.xml#L505) I try to exclude it directly in the bom, this works too but I don't know if we should exclude it there.

Reply:
The `commons-logging-jboss-logging` is already declared in that `pom.xml`

Reply:
Hmm, but why don't we have an error if it's supposed to be excluded?

Reply:
I don't know, this is really weird. This should be handled by the plugin

Reply:
@famod any idea?

Reply:
@aloubyansky, @famod Do you know why `commons-logging` can be in transitive dependencies while it is banned in the `maven-enforcer-plugin` ?

Reply:
No, it catches everything.

There's something weird going on because I don't get any `commons-logging` when doing `mvn dependency:tree` for this module and in particular for `resteasy-client-microprofile` I get:
```
[INFO] +- org.jboss.resteasy:resteasy-client-microprofile:jar:4.7.0.Final:compile
[INFO] |  \- org.jboss.resteasy:resteasy-client-microprofile-base:jar:4.7.0.Final:compile
[INFO] |     +- org.jboss.resteasy:resteasy-client:jar:4.7.0.Final:compile
[INFO] |     |  +- org.jboss.resteasy:resteasy-client-api:jar:4.7.0.Final:compile
[INFO] |     |  \- commons-io:commons-io:jar:2.11.0:compile
[INFO] |     \- org.jboss.resteasy:resteasy-cdi:jar:4.7.0.Final:compile
```

Reply:
I don't think the problem comes from the plug-in but from gradle ignoring it when resolving dependencies. How does maven handles that enforcer rules from dependencies ?

Reply:
The enforcer just checks that the dependencies are not around, it doesn't exclude anything. So if we don't have it in the Maven dependencies, it's because, using the Maven resolver, the dependency is not around. What I don't understand is why Gradle sees it in its dependencies. Maybe it still includes the transitive dependencies of the excluded dependencies?

Reply:
That’s really weird. Some transitive dependencies may bring commons-logging back but I don’t know how